tests/ui/helpers/helpers_test.js

```diff
@@ -17,3 +17,12 @@ describe('displayNumber helper', () => {
     expect(displayNumber(Number.NaN)).toEqual('N/A');
   });
 });
+
+describe('formatNumber helper', () => {
+  test('returns expected values', () => {
+    expect(formatNumber('9873.3321')).toEqual('9,873.33');
+    expect(formatNumber('123123123.53222')).toEqual('123,123,123.53');
+    expect(formatNumber(1 / 0)).toEqual('∞');
+    expect(formatNumber('a string')).toEqual('NaN');
+  });
+});
```

Comment:
Adding a test here shows really good initiative but I don't think it's necessary since it's testing a built-in Javascript method, rather than custom logic. So I think you can remove it (I'm not sure why someone added a test for `displayNumber` but we can leave that for now).